README.md

```diff
@@ -306,6 +306,7 @@ Table of Contents
   * [sanity.io](https://www.sanity.io/) – Hosted backend for structured content with customizable MIT licensed editor built with React. Unlimited projects. 3 users, 2 datasets, 500k API CDN requests, 5GB assets for free per project
   * [sensenet](https://sensenet.com) - API-first headless CMS providing enterprise-grade solutions for businesses of all size. The Developer plan provides 3 users, 500 content items, 3 built-in roles, 25+5 content types, fully accessible REST API, document preview generation and Office Online editing.
   * [GraphCMS](https://graphcms.com/) - Offers free tier for small projects. GraphQL first API. Move away from legacy solutions to the GraphQL native Headless CMS - and deliver omnichannel content API first.
+  * [Umbraco](https://umbraco.com/) - .NET based CMS with flexible editing options. Use the open source version of Umbraco for free and install, setup, and host it yourself.
```

Comment:
The list is for SaaS, it seems to me the free tier is only for self hosted so wouldnt fit the list>

Reply:
Ah yes, sorry about that! Will close.